devpy/cmds/__init__.py

```diff
@@ -1,3 +1,8 @@
-from ._build import build
-from ._test import test
-from ._shell import ipython, python, shell
+from . import meson
+
+# Backward compatibility with older versions
+build = meson.build
```

Comment:
Can we issue a deprecation warning for people still using these directly?

Reply:
Sure, do you know of an easy way to implement that?

Reply:
You can try a module-level __getattr__ (for python 3.7+), that raises the Deprecation/FutureWarning.